Shared/Services/ConfigLoader.swift

```diff
@@ -202,13 +203,27 @@ public struct ConfigLoader {
             throw ConfigLoaderError.CannotSaveOnboardingIsCompleted
         }
     }
-    
+
+    public func hideBackupBanner() throws -> Void {
+        let show = self.saveToUserDefaults(key: "ShowBackupBanner", value: "1")
+
+        if !show {
+            throw ConfigLoaderError.CannotHideBackupBanner
+        }
+    }
+
     public func onboardingIsCompleted() -> Bool {
         let completed = self.getFromUserDefaults(key: "OnboardingIsCompleted")
 
         return completed == "1"
     }
-    
+
+    public func isBackupBannerShown() -> Bool {
```

Comment:
`shouldShowBackupsBanner` or similar I think fits better, as this method is not related to the UI and nothing is going to appear there if we call this right?